<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseForm</class>
 <widget class="QWidget" name="LicenseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>License</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QTextEdit" name="about_textedit">
     <property name="lineWrapMode">
      <enum>QTextEdit::WidgetWidth</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>LicenseForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
